lstchain/image/muon/muon_analysis.py

```diff
@@ -155,28 +204,17 @@ def analyze_muon_event(subarray, event_id, image, geom, equivalent_focal_length,
         Charge "just outside" ring, to check the possible signal extractor bias
     muonparameters : `MuonParametersContainer`
     """
-    # TODO: Several hard-coded quantities that can go into a configuration file
-
-    lst1_tel_id = 1
-    lst1_description = subarray.tels[lst1_tel_id]
 
-    tailcuts = [10, 5]
+    tel_description = subarray.tels[tel_id]
 
     cam_rad = (
-                      lst1_description.camera.geometry.guess_radius() / lst1_description.optics.equivalent_focal_length
+                      tel_description.camera.geometry.guess_radius() / tel_description.optics.equivalent_focal_length
               ) * u.rad
 
-    # some cuts for good ring selection:
-    min_pix = 148  # (8%) minimum number of pixels in the ring with >0 signal
-    min_pix_fraction_after_cleaning = 0.1  # minimum fraction of the ring pixels that must be above tailcuts[0]
-    min_ring_radius = 0.8 * u.deg  # minimum ring radius
-    max_ring_radius = 1.5 * u.deg  # maximum ring radius
-    max_radial_stdev = 0.1 * u.deg  # maximum standard deviation of the light distribution along ring radius
-    max_radial_excess_kurtosis = 1.  # maximum excess kurtosis
-    min_impact_parameter = 0.2  # in fraction of mirror radius
-    max_impact_parameter = 0.9  # in fraction of mirror radius
-    ring_integration_width = 0.25  # +/- integration range along ring radius, in fraction of ring radius (was 0.4 until 20200326)
-    outer_ring_width = 0.2  # in fraction of ring radius, width of ring just outside the integrated muon ring, used to check pedestal bias
+    # some parameters for analysis and cuts for good ring selection:
+    tailcuts, min_pix, min_pix_fraction_after_cleaning, min_ring_radius, max_ring_radius, max_radial_stdev,\
+        max_radial_excess_kurtosis, min_impact_parameter, max_impact_parameter, ring_integration_width,\
+        outer_ring_width, ring_completeness_threshold = update_parameters(good_ring_config, geom.n_pixels)
```

Comment:
I'd just keep them in the dict and use the dict

Reply:
Done